pkg/service/repair/plan.go

```diff
@@ -74,7 +74,9 @@ func newPlan(ctx context.Context, target Target, client *scyllaclient.Client) (*
 		}
 
 		// Update max parallel
-		maxP = max(maxP, MaxRingParallel(ring, target.DC))
+		if ringP := MaxRingParallel(ring, target.DC); maxP < ringP {
+			maxP = ringP
+		}
```

Comment:
I understand it's due to the failing jenkins job. Please revert. Releng will bump go to the correct version (unless we are really in a hurry)

Reply:
done